test-app/tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "resolveJsonModule": true,
+    "skipLibCheck": true,
```

Comment:
what does this do?

Reply:
Test app was failing on some node deps. just tells TS not to check TS in all of the node modules.